src/modbus_ext_common.h

```diff
@@ -90,6 +108,8 @@ namespace ModbusExt // modbus extension protocol common utilities
 
         void EnableEvents();
         void ClearRequest();
+
+        void SendRequest();
```

Comment:
```suggestion
        void SendSingleRequest();
```

И во всех причастных местах

Reply:
Заменил